src/sql/azdata.proposed.d.ts

```diff
@@ -1529,4 +1537,30 @@ declare module 'azdata' {
 		 */
 		isLongString: boolean;
```

Comment:
this property doesn't seem to be correct, what is the criteria to be long? why not just call it positionAtBottom?

Reply:
Need to change this in STS as well. Will do it in a future PR